plugins/headless/hu.bme.mit.gamma.headless.server/src/hu/bme/mit/gamma/headless/server/OpenApiWebServer.java

```diff
@@ -142,23 +148,34 @@ public void start(Promise<Void> future) {
 
 				// Gets the log file from a workspace
 				routerFactory.operation("getLogs").handler(routingContext -> {
+					
+					logger.log(Level.INFO, ANSI_YELLOW + "Operation \"getLogs\" has started." + ANSI_RESET);
+					
 					// Getting parameters from path
 					RequestParameters params = routingContext.get(PARSED_PARAMETERS);
 					String workspace = params.pathParameter(WORKSPACE).getString();
 					// Path of the log file
 					String logPath = FileHandlerUtil.getProperty(DIRECTORY_OF_WORKSPACES_PROPERTY_NAME) + File.separator
 							+ workspace + File.separator + ".metadata" + File.separator + ".log";
 					boolean success = false;
-					try {
-						// Getting the log file
-						FileInputStream inputLog = new FileInputStream(logPath);
-						inputLog.close();
-						success = true;
-					} catch (FileNotFoundException e) {
-						e.printStackTrace();
-					} catch (IOException e) {
-						e.printStackTrace();
+					if(!loggingToFile) {
+						try {
+							// Getting the log file
+							FileInputStream inputLog = new FileInputStream(logPath);
+							inputLog.close();
+							success = true;
+						} catch (FileNotFoundException e) {
```

Comment:
Here and everywhere else later: if you just catch the `FileNotFoundException` and in the subsequent block, the `IOException`, you treat them the same (print the stack trace) then I believe two separate catch blocks are unnecessary, and just increase code noise (leave the `IOException` block and delete the `FileNotFoundException` block).